controls/radcoloreditor/features/color-mode.md

```diff
@@ -14,15 +14,35 @@ The __RadColorEditor__ control allows supports the most popular color models. Yo
 
 The __ColorMode__ property is an enumeration of type __ColorMode__ and it exposed the following members:				
 
+#### __[XAML] Example 1: Set the color model to RGB__
+{{region radcoloreditor-features-color-mode_0}}
+	<telerik:RadColorEditor x:Name="colorEditor" SelectedColor="Red" ColorMode="RGB"/>
+{{endregion}}
+
 * __RGB__ - sets the color model to[RGB(Red, Green and Blue)](http://en.wikipedia.org/wiki/RGB_color_model)  
 	![radcoloreditor-features-rgb](images/radcoloreditor-features-rgb.png)
 
+#### __[XAML] Example 2: Set the color model to HLS__
+{{region radcoloreditor-features-color-mode_1}}
+	<telerik:RadColorEditor x:Name="colorEditor" SelectedColor="Green" ColorMode="HLS"/>
+{{endregion}}

```

Comment:
Move all code snippets related to list items with a single tab forward. This way the code snippet will become part of the list item. Otherwise it brakes the list and goes out side of it.

So, move the entire block marked with this comment with a single tab forward. Do this for all code snippets.